Source/NSScrollView.m

```diff
@@ -1213,10 +1213,15 @@ - (void) tile
 
       /** Vertically expand the scroller by 1pt on each end */
       if (overlapBorders)
-	{
+        {
 	  vertScrollerRect.origin.y -= 1;
 	  vertScrollerRect.size.height += 2;
 	}
+      else if (_hasHeaderView || _hasCornerView)
```

Comment:
What if we have overlapBorders and a header view? Not sure what should happen, but for now this is OK.

Reply:
I don't know either.